tests/scripts/thread-cert/v1_2_LowPower_7_2_01_ForwardTrackingSeries.py

```diff
@@ -0,0 +1,340 @@
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+import unittest
+
+from config import ADDRESS_TYPE
+from mle import LinkMetricsSubTlvType, TlvType
+from pktverify import consts
+from pktverify.null_field import nullField
+from pktverify.packet_verifier import PacketVerifier
+
+import thread_cert
+
+LEADER = 1
+SED_1 = 2
+SSED_1 = 3
+
+SERIES_ID = 1
+SERIES_ID_2 = 2
+POLL_PERIOD = 2000  # 2s
+
+
+class LowPower_7_2_01_ForwardTrackingSeries(thread_cert.TestCase):
+    TOPOLOGY = {
+        LEADER: {
+            'version': '1.2',
+            'name': 'LEADER',
+            'mode': 'rdn',
+            'panid': 0xface,
+            'allowlist': [SED_1, SSED_1],
+        },
+        SED_1: {
+            'version': '1.2',
+            'name': 'SED_1',
+            'mode': '-',
+            'panid': 0xface,
+            'allowlist': [LEADER],
+        },
+        SSED_1: {
+            'version': '1.2',
+            'name': 'SSED_1',
+            'mode': '-',
+            'panid': 0xface,
+            'allowlist': [LEADER],
+        }
+    }
+    """All nodes are created with default configurations"""
+
+    def test(self):
+        self.nodes[LEADER].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
+
+        self.nodes[SED_1].set_pollperiod(POLL_PERIOD)
+        self.nodes[SED_1].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[SED_1].get_state(), 'child')
+
+        self.nodes[SSED_1].set_csl_period(consts.CSL_DEFAULT_PERIOD)
+        self.nodes[SSED_1].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[SSED_1].get_state(), 'child')
+
+        leader_addr = self.nodes[LEADER].get_ip6_address(ADDRESS_TYPE.LINK_LOCAL)
+
+        # Step 3 - Verify connectivity by instructing each device to send an ICMPv6 Echo Request to the DUT
+        self.assertTrue(self.nodes[SED_1].ping(leader_addr, timeout=POLL_PERIOD))
+        self.assertTrue(self.nodes[SSED_1].ping(leader_addr, timeout=2 * consts.CSL_DEFAULT_PERIOD))
+        self.simulator.go(5)
+
+        # Step 4 - SED_1 requests a Forward Tracking Series by sending a Link Metrics Management Request
+        # Forward Series Flags = 0x04:
+        # - Bit 2 = 1, MAC Data Request
+        # - Bits 0,1 & 3-7 = 0
+        # Concatenation of Link Metric Type ID Flags = 0x00:
+        # - Item1: (0)(1)(000)(000) = 0x40
+        # -- E = 0
+        # -- L = 1
```

Comment:
does it conflict with Test Plan r08?

It seems that for `L 0/1` we discussed, Test Plan indicates the value of PDU (Metrics Enum 0) is of 1 byte (Step 4 ~8) if L bit is 0, and it is 4 bytes. (Step 11 ~ 14) if L bit is 1;  however OT would always report 4 bytes PDU, regardless of `L` bit. I am afraid existing OT implementation may fail this test case.

Thoughts?

Reply:
The test case is problematic as I mentioned in PR description: https://github.com/openthread/openthread/pull/5608#issue-498391452. They haven't completed Link Metrics cases.